drush/policy.drush.inc

```diff
@@ -1,38 +0,0 @@
-<?php
-
-/*
- * Customize this file as desired. See https://github.com/drush-ops/drush/blob/8.x/examples/policy.drush.inc for documentation.
- */
-
-/**
- * Implements drush_hook_COMMAND_validate().
- *
- * Encourage folks to use `composer` instead of Drush pm commands
- */
-function drush_policy_pm_updatecode_validate() {
-  return _deny_message();
```

Comment:
This functionality is not present in the replacement `Drush\Commands\PolicyCommands` - should we not continue to prevent this?

Reply:
There is no pm-updatecode anymore so no need. Of course people still use Drush8 but thsoe folks are generally not starting brand new projects. This project is a starter-kit so we dont have to worry about existing sites.

Reply:
Same for `pm-update` and `pm-download` ...

Reply:
Ah - I should have checked the repo, I just did a quick check on _drushcommands.com_ !